### PR TITLE
docs: Convert docbook content to readthedocs and reStructuredText format

### PR DESCRIPTION
**Pull request by haoweiqiu**
This PR partially resolves issue #665 and #666.
The ReadtheDocs link is: https://pcp-docs.readthedocs.io

### CODE REVIEW

File: docs/pcp-programmers-guide.rst [hunk 1]
```diff
@@ -0,0 +1,7966 @@
+========================================
+Performance Co-Pilot™ Programmer's Guide
+========================================
+
+.. _id5178752:
```
Comment (andreasgerstmayr) [inline docs/pcp-programmers-guide.rst:5]: What are these markers?

File: docs/pcp-users-and-administrators-guide.rst [hunk 2]
```diff
@@ -0,0 +1,7051 @@
...
+
+1. Log in as ``root`` (the superuser).
+
+2. Change to the PMDA's directory as shown in the following example:
+
+   ::
+
+      cd ${PCP_PMDAS_DIR}/cisco
+
+3. PMDPerformance Metrics DomainPMDIn the unlikely event that you wish
```
Comment (andreasgerstmayr) [inline docs/pcp-users-and-administrators-guide.rst:1765]: Something's off here, where is `PMDPerformance Metrics DomainPMD` coming from? Did you write a script to automatically convert the xml to rst (https://github.com/performancecopilot/pcp/blob/master/books/PCP_UAG/pcp-users-and-administrators-guide.xml#L1847)? (Good idea btw :+1:)

File: docs/pcp-programmers-guide.rst (hunk 1 above)
Comment (haoweiqiu) [reply docs/pcp-programmers-guide.rst:5]: The markers were created for link reference, however, it seems that the markers don't work well in Read the Docs. I would remove the markers soon.

File: docs/pcp-users-and-administrators-guide.rst (hunk 2 above)
Comment (haoweiqiu) [reply docs/pcp-users-and-administrators-guide.rst:1765]: Oops, it seems that there is a conversion error here. Actually, a converting tool was used for this task. I would need some extra time to review and improve the docs. Thanks for the notice!

**Comment by natoscott** [comment]
Discussing with @arzoo14 he's planning to tackle this differently, making separate rst files for every chapter.

**Pull request closed without merging**